sql1.txt

```diff
@@ -3,41 +3,70 @@ table.  For example, to see the schema for the Products table, you could do:
 
 SELECT * FROM Products LIMIT 5;
 
+#####################
+
+
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
 
+SELECT * FROM Customers ORDER BY CustomerName LIMIT 10;
+
+#####################
+
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
 
+SELECT ProductName, Price FROM Products WHERE Price < 20;
+
+#####################
+
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
 
+SELECT * FROM Employees WHERE LastName LIKE 'C%';
+
+#####################
 
 4. Retrieve the order ID and customer name for all orders where the customer name starts with A.
 Here you will have to do a join of the Orders table and the Customers table.  Paste your SQL
 statement below.
 
+SELECT OrderID, CustomerName FROM Customers 
+INNER JOIN Orders ON Orders.CustomerID = Customers.CustomerID 
+WHERE CustomerName LIKE 'A%';
+
+#####################
 
 5. Retrieve the list of customers, ordered by customer name, along with their order IDs.  Note
 that not every customer has an order, but be sure that you include those customers without orders
 in the results of the query.  Paste your SQL statement below.
 
+SELECT CustomerName, OrderID FROM Customers
+LEFT JOIN Orders ON Orders.CustomerID = Customers.CustomerID
+ORDER BY CustomerName;
```

Comment:
It is a bit unclear to me if this questions expectation is to grab **all columns in `Customers`** or just the `CustomerName`, as you have done.

However, just a note that you can grab all customer columns and the order ID by changing this slightly:

```suggestion
SELECT Customers.*, OrderID FROM Customers
LEFT JOIN Orders ON Orders.CustomerID = Customers.CustomerID
ORDER BY CustomerName;
```